initiatives/submit-a-request/index.html

```diff
@@ -0,0 +1,371 @@
+    {% endif %}
+
+    {% if initiative.content %}
+    <section class="initiative_description block block__flush-top">
+        {{ initiative.content | safe }}
+
+        <!-- Extra Content. Remove once content updates. -->
+        <ul class="short-desc">
+            <li>Step 1: Fill out the form</li>
+            <li>Step 2: Generate a PDF</li>
+            <li>Step 3: Email, fax or mail the PDF</li>
+        </ul>
+    </section>
+    {% endif %}
+
+    <!-- Start Form Module. -->
+    <section class="block block__bg block__border block__flush-sides">
+        <form class="form-l js-validate-filters">
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="foia-request_first-name">
+                    First name
+                </label>
+                <input class="input__long"
+                       id="foia-request_first-name"
+                       required
+                       type="text">
+            </div>
+
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="foia-request_last-name">
+                    Last name
+                </label>
+                <input class="input__long"
+                       id="foia-request_last-name"
+                       required
+                       type="text">
+            </div>
+            <fieldset class="u-reset">
+                <div class="form-l_col form-l_col-1 u-mb0">
+                    <legend class="form-label-header">
+                        Document dates
+                    </legend>
+                </div>
+                <div class="form-l_col form-l_col-1-2">
+                    <label class="micro-copy micro-copy__large" for="foia-request_starting-date">
+                        From:
+                    </label>
+                    <input class="input__long"
+                           id="foia-request_starting-date"
+                           placeholder="mm/dd/yyyy"
+                           type="text">
+                </div>
+                <div class="form-l_col form-l_col-1-2 input__long">
+                    <label class="micro-copy micro-copy__large" for="foia-request_ending-date">
+                        To:
+                    </label>
+                    <input class="input__long"
+                           id="foia-request_ending-date"
+                           placeholder="mm/dd/yyyy"
+                           type="text">
+                </div>
+            </fieldset>
+
+            <div class="form-l_col form-l_col-1">
+                <label for="foia-request_request-desc">
+                    <span class="form-label-header">
+                        Request description
+                    </span><br/>
+                    <span class="micro-copy micro-copy__large">
+                        Describe the records you seek from the CFPB. Review our FAQs for additional guidance
```

Comment:
Should we link to the FAQ page from here? Probably with a `target="_blank"` attr so the user doesn't lose any entered info.